packages/bundler-plugin-core/src/index.ts

```diff
@@ -121,53 +118,87 @@ export function sentryUnpluginFactory({
       })
     );
 
-    if (options.injectRelease && releaseName) {
+    if (!options.release.inject) {
+      logger.debug("Release injection disabled via `release.inject`. Will not inject.");
+    } else if (!options.release.name) {
+      logger.warn("No release name provided. Will not inject release.");
+    } else {
       const injectionCode = generateGlobalInjectorCode({
-        release: releaseName,
+        release: options.release.name,
         injectBuildInformation: options._experiments.injectBuildInformation || false,
       });
-
       plugins.push(releaseInjectionPlugin(injectionCode));
     }
 
-    if (options.sourcemaps?.assets) {
-      plugins.push(debugIdInjectionPlugin());
-    }
-
-    if (releaseName) {
+    if (!options.release.name) {
+      logger.warn("No release name provided. Will not manage release.");
```

Comment:
l: Hmm not sure how we're communitcating this plugin in the docs/readme in the future, so feel free to disregard if you think this makes sense. But IMO as a user I'd be confused what this message means. Let's make it clear that we're talking about releases on Sentry.

I'm not too sure about a better wording but what about something along these lines:

```suggestion
      logger.warn("No release name provided. Will not create release on Sentry.");
```

or

```suggestion
      logger.warn("No release name provided. Will not create a Sentry release.");
```

Reply:
Oh I just realized, you addressed this in https://github.com/getsentry/sentry-javascript-bundler-plugins/pull/240 - will mark this as resolved then!